client/windows/CCastleInterface.cpp

```diff
@@ -632,9 +632,9 @@ void CCastleBuildings::addBuilding(BuildingID building)
 		{
 			//reset animation
 			if(structures.size() == 1)
-				buildingRect->stateCounter = 0; // transparency -> fully visible stage
+				buildingRect->stateTimeCounter = 0; // transparency -> fully visible stage
 			else
-				buildingRect->stateCounter = 16; // already in fully visible stage
+				buildingRect->stateTimeCounter = 500; // already in fully visible stage

```

Comment:
1. Move S1_TRANSP and other consts outside of class method
2. 
```suggestion
				buildingRect->stateTimeCounter = S1_TRANSP; // already in fully visible stage
```